.env

```
@@ -13,46 +13,46 @@ if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
-            export VAULT_TOKEN_FILE
-            break
-        fi
-    done
+if ! command -v vault >/dev/null 2>&1; then
+    echo "Vault CLI is required but not found in PATH" >&2
+    return 1
 fi
 
-SECRET_DEFS="${VAULT_SECRET_PATHS:-$DEFAULT_VAULT_SECRET_DEFS}"
-REQUIRED_VARS="${VAULT_REQUIRED_VARS:-$DEFAULT_VAULT_REQUIRED_VARS}"
+source <(
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The source <( key has incorrect delimiter

(IncorrectDelimiter)